xtask/README.md

```diff
@@ -17,10 +17,12 @@ Usage: xtask <COMMAND>
 
 Commands:
   check-fmt      Run cargo fmt on extendr
+  fmt            Run `cargo fmt` on extendr crates
   r-cmd-check    Run R CMD check on {extendrtests}
   doc            Generate documentation for all features
   msrv           Check that the specified rust-version is MSRV
   devtools-test  Run devtools::test() on {extendrtests}
+  document       Generate wrappers by `rextendr::document`
```

Comment:
Let's be consistent here, if you apply ``` `` ``` to code, change all other cases in the list. Also, when mentioning a function, don't forget a `()`

Reply:
Something like this
```suggestion
  document       Generate wrappers with `rextendr::document()`
```